src/services/ReportService.ts

```diff
@@ -326,14 +326,37 @@ const updateReportPhoto = async (reportId: number, photoId: number, caption: str
   });
 };
 
+/**
+ * Delete multiple photos for a report
+ */
+const deleteReportPhotos = async (reportId: number, photosId: number[]): Promise<(Response | string)[]> => {
+  const deletePhotoPromises = photosId.map((photoId) => deleteReportPhoto(reportId, photoId));
+  try {
+    const promiseResponses = await Promise.allSettled(deletePhotoPromises);
```

Comment:
there's a util in `src/services/utils` called `getPromisesResponse` which you can reuse here